code/modules/mob/living/simple_animal/friendly/drone/interaction.dm

```diff
@@ -49,11 +48,8 @@
 			return
 		to_chat(user, "<span class='notice'>You pick [src] up.</span>")
 		drop_all_held_items()
-		var/obj/item/clothing/head/drone_holder/DH = new /obj/item/clothing/head/drone_holder(src)
-		DH.updateVisualAppearence(src)
-		DH.drone = src
+		var/obj/item/clothing/head/mob_holder/drone/DH = new(get_turf(src), src)
```

Comment:
Cogscarabs cannot be held for balance reasons, so make sure to keep that.

Reply:
already a thing in pr.